kerlescan/view_helpers.py

```diff
@@ -9,6 +9,85 @@
 from kerlescan.rbac_service_interface import get_roles
 from kerlescan.exceptions import HTTPError
 
+pagination_link_template = "%s?limit=%s&offset=%s&order_by=%s&order_how=%s"
+
+
+def _create_first_link(path, limit, offset, count, order_by, order_how):
```

Comment:
I would add a small comment to the beginning of these methods with an example of the returned string (`/api/item-service/v1/items?limit=20&offset=0&order_by=captured_date&order_how=asc` or whatever), just to make it easier to understand.